5.3/Encrypted logo.py

```diff
@@ -0,0 +1,44 @@
+""" The program open logo file in binary style.
+The binary file is encrypted and there are some secret lines.
+Secret lines are the once that end with !, their len is >=5 and
+they include just lower letters.
+The program find and print secrete lines.
```

Comment:
This documentation is good, but its redundant. document the relevant parts not the head of the page